data/transactions/logic/TEAL_opcodes.md

```diff
@@ -790,7 +790,7 @@ When A is a uint64, index 0 is the least significant bit. Setting bit 3 to 1 on
 
 ## json_ref r
 
-- Opcode: 0x5f {string return type}
+- Opcode: 0x5f {uint8 return type}
```

Comment:
nit: I am confused here, for what I remembered it is returning a string (or byte slice) of json object utf-8 encoded. Does it mean uint8 slice/array here?

Reply:
The thing in the braces is the immediate opcode argument to the json_ref opcode. The json_ref opcode's encoding is two bytes, the first byte is the opcode itself, and the second byte is a number that reflects the type that the caller thinks it's going to find, like `JSONString`, `JSONUint64`.  It has always been a byte, never a string.  Just a doc bug.

Reply:
oh boy, I am acting dumb here. Resolving.